SubmissionTest.java

```diff
@@ -603,6 +603,272 @@ public void testGetCostumeCountForSpriteValid()
     }
     
     /**
+     * Test getControlBlocksForStage method.
+     */
+    @Test
+    public void testGetControlBlocksForStage()
+    {
+        setUpSubmission(submissions[1]);
+        submissions[1].countBlockCategoriesForStage();
+
+        int expected = 1;
+        int actual = submissions[1].getControlBlocksForStage();
+        assertEquals("should be equal", expected, actual);
```

Comment:
Yes it probably could. But I thought we had been formatting all the unit tests this way? I think it might help for documentation to show that one is our preset expected value and one is the actual value returned from the tested method. BUT - if you think it is clear/documented enough as the one-liner, I can go in and change all of them accordingly. Let me know what you think! Thanks!